geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/io/LayerReaderFactory.scala

```diff
@@ -121,33 +110,20 @@ abstract class FilteringLayerReaderWrapper()
 
   def query(
     keyType: String,
-    valueType: String,
     layerName: String,
     zoom: Int,
     queryGeometryString: String,
     queryIntervalStrings: ArrayList[String]
   ): (JavaRDD[Array[Byte]], String) = {
     val id = LayerId(layerName, zoom)
 
-    (keyType, valueType) match {
-      case ("SpatialKey", "Tile") => {
-        val spatialQuery = getSpatialQuery(queryGeometryString)
-
-        val layer = layerReader.query[SpatialKey, Tile, TileLayerMetadata[SpatialKey]](id)
-        val query = spatialQuery match {
-          case Some(point: Point) => layer.where(Contains(point))
-          case Some(polygon: Polygon) => layer.where(Intersects(polygon))
-          case Some(multi: MultiPolygon) => layer.where(Intersects(multi))
-          case None => layer
-          case _ => throw new Exception(s"Unsupported Geometry $spatialQuery")
-        }
-        PythonTranslator.toPython(query.result)
-      }
-      case("SpatialKey", "MultibandTile") => {
+    keyType match {
+      case "SpatialKey" => {
         val spatialQuery = getSpatialQuery(queryGeometryString)
 
         val layer = layerReader.query[SpatialKey, MultibandTile, TileLayerMetadata[SpatialKey]](id)
         val query = spatialQuery match {
+          case Some(point: Point) => layer.where(Contains(point))
```

Comment:
@echeipesh Will we still be able to query `Point`s within a `MultibandTile`? Because the original code only matched against `Point`s when `V` was a `Tile`.